pkg/bundle/sync.go

```diff
@@ -31,6 +34,11 @@ import (
 
 	trustapi "github.com/cert-manager/trust-manager/pkg/apis/trust/v1alpha1"
 	"github.com/cert-manager/trust-manager/pkg/util"
+	jks "github.com/pavlo-v-chernykh/keystore-go/v4"
+)
+
+const (
+	defaultJksPassword = "changeit"
```

Comment:
I don't use the JKS format much, but I imagine that we don't really care about the password for this specific keystore (it will never contain private key material)

Is there a possibility to set the password to `""`? 

Otherwise, the hardcoded `"changeit"` may be fine as long as we document it

Reply:
Pretty sure it needs some kind of password, didn't try to see if an empty string was acceptable.

As far as I'm aware, java apps kinda expect `changeit` as a default - I think this is the least painful thing to set it to.
You're right though, there should be a comment at least to explain.

Reply:
Aah great, if changeit is a defacto standard then people won't be surprised! 👌

Reply:
> Aah great, if "changeit" is a defacto standard then people won't be surprised!

That's my understanding from poking around, but I'm no java dev :)

Reply:
I've seen at least one major system using certs with Java that used `changeit` as a default. I'm happy to have that be the default here!